--- a/vpm	2020-02-18 19:04:21.171793956 +0300
+++ b/vpm	2019-12-25 13:05:42.020392311 +0300
@@ -192,7 +192,7 @@
   echo "listrepos (lr)                 - List configured repositories"
   echo "repolist (rl)                  - Alias for listrepos"
   t "xbps-query -v -L"
-  echo "addrepo (ar) <ARGS>            - Add an additional repository"
+  echo "addrepo <ARGS>            - Add an additional repository"
   t "xbps-install <ARGS>"
   echo "info (in) <pkg>                - Show information about <package>"
   t "xbps-query-v -R"
@@ -402,7 +402,7 @@
     echo
   ;;
 
-  addrepo|ar)
+  addrepo)
     shift
     rootcheck
     for repo in "$@"; do
@@ -536,6 +536,7 @@
       exit 0 
     fi
     rmsg "$ret" "[xbps-install -Suv] return code: $ret"
+    exit "$ret"
   ;;
 
   search|s)
